catalog/app/containers/Bucket/PackageTree.js

```diff
@@ -189,9 +189,9 @@ function PkgCode({ data, bucket, name, revision, path }) {
           contents: dedent`
             import quilt3
             # browse
-            quilt3.Package.browse("${name}"${hashPy}, registry="s3://${bucket}")
-            # download
-            quilt3.Package.install("${nameWithPath}"${hashPy}, registry="s3://${bucket}", dest=".")
+            p = quilt3.Package.browse("${name}"${hashPy}, registry="s3://${bucket}")
+            # download (be mindful of large packages)
+            p = quilt3.Package.install(${nameWithPath}${hashPy}, registry="s3://${bucket}", dest=".")
```

Comment:
```suggestion
            quilt3.Package.install(${nameWithPath}${hashPy}, registry="s3://${bucket}", dest=".")
```
`Package.install()` returns nothing since a40d4b05ffa8a7cd44451dfee3aaddb88c70304c.